R/compile_report.R

```diff
@@ -75,8 +75,11 @@ compile_report <- function(file, quiet = FALSE, factory = getwd(),
   ## report_outputs/; this folder will be named after the input report,
   ## e.g. `foo_date/compiled_[date]_[time]/`, and will include `params` if
   ## provided (see point 3).
-  
+
   validate_factory(factory)
+  
+  # This is used for loggin later in the function 
+  log_entry <- as.list(c(as.list(environment())))
```

Comment:
Note: this will capture only named arguments, not `...` ; should be fine as we now capture parameters through `params`, and passively pass `...` to `rmarkdown::render`

Reply:
It is capturing `...` i was testing it manually with the `other` param below - I'll add the test.

Reply:
nevermind!